content/docs/use-cases/data-registries.md

```diff
@@ -22,22 +22,18 @@ _middleware_ between ML projects and cloud storage. Here are its advantages:
 - Reusability: reproduce and organize _feature stores_ with a simple CLI
   (`dvc get` and `dvc import` commands, similar to software package management
   systems like `pip`).
-- Persistence: the DVC registry-controlled
-  [remote storage](/doc/command-reference/remote) (e.g. an S3 bucket) improves
-  data security. There are less chances someone can delete or rewrite a model,
-  for example.
-- Storage optimization: track data
-  [shared](/doc/use-cases/sharing-data-and-model-files) by multiple projects
-  centralized in a single location (with the ability to create distributed
-  copies on other remotes). This simplifies data management and optimizes space
-  requirements.
-- Data as code: leverage Git workflow such as commits, branching, pull requests,
-  reviews, and even CI/CD for your data and models lifecycle. Think "Git for
-  cloud storage", but without ad-hoc conventions.
-- Security: registries can be setup to have read-only remote storage (e.g. an
-  HTTP location). Git versioning of
-  [DVC metafiles](/doc/user-guide/dvc-files-and-directories) allows us to track
-  and audit data changes.
+- Persistence: [remote storage](/doc/command-reference/remote) (e.g. an S3
```

Comment:
we'll do another iteration on the data registries soon (hopefully) ... I think it would improve readability if you highlight certain key phrases in each item. People do not read long lists as they read text.

Reply:
Agree. Thanks

Reply:
p.s. in fact there's #1737 for this already.